blocks/footer/footer.css

```diff
@@ -16,5 +17,125 @@ footer .footer p {
 
 footer .footer-slogan {
   font-size: var(--footer-font-size-slogan);
+  padding-bottom: 40px;
+  font-style: italic;
+}
+
+footer li {
+  display: inline-block;
+  margin-right: 1rem;
+}
+
+footer li a:any-link {
+  color: var(--text-color-on-black-background);
+}
+
+footer li a:hover {
+  color: var(--link-hover-color);
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+
+.footer-container .left-side {
+  align-self: flex-start;
+}
+
+.footer-container .right-side {
```

Comment:
let's also make a hover for the social icons on the right.

Reply:
done